app/parse.py

```diff
@@ -1,4 +1,8 @@
-from dataclasses import dataclass
+from dataclasses import dataclass, astuple, fields
+import csv
```

Comment:
it is better add an empty line between different imports, like included and imported